src/components/new-form/form-controller/index.tsx

```
@@ -0,0 +1,82 @@
+  render,
+  size,
+  ...controllerProps
+}: FormControllerProps<TFieldValues, TName, TTransformedValues>) {
+  return (
+    <Controller
+      {...controllerProps}
+      render={({ field, fieldState }) => {
+        const id = field.name;
+        // We are inside a render function so it's fine
+        // eslint-disable-next-line react-hooks/rules-of-hooks
+        const fieldCtx = useMemo(
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)